src/modules/service-catalog/components/service-catalog-list-item/Search.tsx

```diff
@@ -43,6 +81,9 @@ export const Search = ({
         )}
         onChange={(event) => onChange(event.target.value)}
       />
+      <StyledEndIconContainer>
```

Comment:
Nit: maybe we can apply the styles directly to the `SearchEndIcon` component itself instead of creating a container component, but it is a minor thing, we can also keep the container